vyper/semantics/types/utils.py

```diff
@@ -104,7 +104,12 @@ def _failwith(type_name):
     if node.id not in namespace:
         _failwith(node.node_source_code)
 
-    return namespace[node.id]
+    typ_ = namespace[node.id]
+    if hasattr(typ_, "from_annotation"):
```

Comment:
do any types not have this method?

Reply:
Yes, only bytestrings an subscriptable types have this method.

Reply:
ah, i see